doc/source/reference/commands/status.rst

```diff
@@ -147,3 +149,39 @@ values:
    * - ``version``
      - The version of the Groonga process.
      - ``5.0.7``
+
+   * - ``features``
+     - .. versionadded:: 10.0.1
+     
+       The list of features that Groonga has and whether they are enabled or not.
```

Comment:
is it common sense that true means "enabled" and false means "disabled"?
This is just my question...

Reply:
I think, for developers or users of Groonga,  it is common sense, or at least it makes sense.

Reply:
It is very interesting. I shall remember! thank you.